.buildkite/nightly-benchmarks/scripts/Dockerfile-hpu-benchmarks

```diff
@@ -0,0 +1,39 @@
+# Use the base image
```

Comment:
does it work with buildkite?

Reply:
Disabled using buildkite as it requires pvt token to upload

Reply:
So it runs with Jenkins? If so please move to the jenkins dir